src/ts-node.ts

```diff
@@ -101,12 +113,10 @@ export function tsPath(root: string, orig: string | undefined): string | undefin
     registerTSNode(root)
     const tsconfig = tsconfigs[root]
     if (!tsconfig) return orig
-    const {rootDir, rootDirs, outDir} = tsconfig.compilerOptions
-    const rootDirPath = rootDir || (rootDirs || [])[0]
-    if (!rootDirPath || !outDir) return orig
+    const {rootDir, rootDirs, outDir: lib} = tsconfig.compilerOptions // ./lib
+    const src = rootDir || (rootDirs || [])[0] // ./src
+    if (!src || !lib) return orig
     // rewrite path from ./lib/foo to ./src/foo
-    const lib = path.join(root, outDir) // ./lib
-    const src = path.join(root, rootDirPath) // ./src
```

Comment:
`parseJsonConfigFileContent()` resolves `rootDir`, `rootDirs`, and `outDir` to their full paths, so no need to join their path with the root.